integreat_cms/static/src/js/poi-box.ts

```diff
@@ -3,11 +3,11 @@ import { getCsrfToken } from "./utils/csrf-token";
 
 type FormResponse = { success: boolean; poi_address_container: string };
 
-const toggleContactFieldBox = (show: boolean) => {
+const toggleContactFieldBox = () => {
```

Comment:
```suggestion
const searchContactFieldBox = () => {
```

Since there is not toggling going on anymore :)